pkg/webui/components/location-form/index.js

```diff
@@ -80,8 +80,14 @@ class LocationForm extends Component {
 
     this.setState({ error: '' })
 
+    const intValues = {
+      latitude: parseInt(values.latitude),
+      longitude: parseInt(values.longitude),
+      altitude: parseInt(values.altitude),
```

Comment:
This cannot work. Location values are floats, not ints, so you'd loose the decimals. Moreover, the correct spot to fix this is in the input component itself, by checking whether the input type is `number` and automatically parsing the value in that case.